Package.swift

```diff
@@ -9,7 +9,7 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/tadija/AEXML", .upToNextMinor(from: "4.4.0")),
-        .package(url: "https://github.com/kylef/PathKit", .upToNextMinor(from: "1.0.0")),
+        .package(url: "https://github.com/michaeleisel/PathKit", .branch("master")),
```

Comment:
Feel like this may cause issues with people who depend on `XcodeProj` _and_ `PathKit` (independently) as it may cause conflicts. Maybe spin off a new library with a new name instead of `PathKit` to avoid conflicts? Either that or try and convince the maintainer to hand it over 😄

Reply:
Also best to pin to a particular release or commit I think, rather than `master`

Reply:
This will indeed cause conflicts, if this is to happen the fork needs a separate module name.

As an aside, since PathKit is not maintained (and is rubbish IMO) may I suggest [my pathing library](https://github.com/mxcl/Path.swift)? It's better.

Reply:
how does its speed compare to this fork?

Reply:
I'm sure the speed isn’t as good as your fork. Though I would want benchmarks on how the pathing library impacts the speed of this project in use personally. Ultimately I am not involved enough in this project to deserve a say on this PR.

Reply:
i made specific changes to PathKit to fix bottlenecks for my usage of XcodeGen. i don't know if PathKit, with or without my changes, is faster or slower than other libraries. personally, and i'm not one of the main contributors either, but i'm happy to use anything that is as fast. i will go ahead with this fork unless you want to do perf tests

Reply:
> Maybe spin off a new library with a new name instead of PathKit to avoid conflicts? Either that or try and convince the maintainer to hand it over

I agree with @dflems here. What about giving it a new name to avoid conflicts? 

In regards to using another pathing library, I'd avoid it unless there're clear advantages over the PathKit's fork.